src/json_interface.jl

```diff
@@ -101,32 +102,62 @@ save model constructor, model parameters, and other information to a JSON file.
 * `errors` : the fitting / test errors computed during the fitting 
 * `verbose` : print information about the saving process     
 """
-function save_model(model, filename; 
```

Comment:
as a general rule you should try to avoid introducing white-space changes. This makes it very hard for me to review and see what you have changed and what has remained the same.

Reply:
Sorry for that, I'll avoid these changes